<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automated Blend Machine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Pictures/Infineum-Colour-Logo.png</normaloff>../Pictures/Infineum-Colour-Logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="stage_lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stage:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_preset">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Preset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_run">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>16</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>330</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="procd_lbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Procedure Steps</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="beakers_tabwid">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Beaker1_tab">
     <attribute name="title">
      <string>Beaker 1</string>
     </attribute>
     <widget class="QTabWidget" name="subtabs1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="weight1">
       <attribute name="title">
        <string>Weight</string>
       </attribute>
       <widget class="QGraphicsView" name="grph_weight1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mix1">
       <attribute name="title">
        <string>Mixing Torque</string>
       </attribute>
       <widget class="QDial" name="dial_mix1">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_mix1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_mix1">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Torque</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_mix1">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_mix1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hot1">
       <attribute name="title">
        <string>Hot Plate Temperature</string>
       </attribute>
       <widget class="QDial" name="dial_hot1">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_hot1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_hot1">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Heat</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_hot1">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_hot1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="beaker1_lbl">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Beaker 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Beaker2_tab">
     <attribute name="title">
      <string>Beaker 2</string>
     </attribute>
     <widget class="QLabel" name="beaker2_lbl">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Beaker 2</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="subtabs2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="weight2">
       <attribute name="title">
        <string>Weight</string>
       </attribute>
       <widget class="QGraphicsView" name="grph_weight2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mix2">
       <attribute name="title">
        <string>Mixing Torque</string>
       </attribute>
       <widget class="QDial" name="dial_mix2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_mix2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_mix2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Torque</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_mix2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_mix2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hot2">
       <attribute name="title">
        <string>Hot Plate Temperature</string>
       </attribute>
       <widget class="QDial" name="dial_hot2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_hot2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_hot2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Heat</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_hot2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_hot2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Beaker3_tab">
     <attribute name="title">
      <string>Beaker 3</string>
     </attribute>
     <widget class="QTabWidget" name="subtabs3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="weight3">
       <attribute name="title">
        <string>Weight</string>
       </attribute>
       <widget class="QGraphicsView" name="grph_weight3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mix3">
       <attribute name="title">
        <string>Mixing Torque</string>
       </attribute>
       <widget class="QDial" name="dial_mix3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_mix3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_mix3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Torque</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_mix3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_mix3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hot3">
       <attribute name="title">
        <string>Hot Plate Temperature</string>
       </attribute>
       <widget class="QDial" name="dial_hot3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_hot3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_hot3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Heat</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_hot3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_hot3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="beaker3_lbl">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Beaker 3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Beaker4_tab">
     <attribute name="title">
      <string>Beaker 4</string>
     </attribute>
     <widget class="QTabWidget" name="subtabs4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="weight4">
       <attribute name="title">
        <string>Weight</string>
       </attribute>
       <widget class="QGraphicsView" name="grph_weight4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mix4">
       <attribute name="title">
        <string>Mixing Torque</string>
       </attribute>
       <widget class="QDial" name="dial_mix4">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_mix4">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_mix4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Torque</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_mix4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_mix4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hot4">
       <attribute name="title">
        <string>Hot Plate Temperature</string>
       </attribute>
       <widget class="QDial" name="dial_hot4">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spin_hot4">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl0_hot4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Heat</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl1_hot4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjust Duration</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="grph_hot4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>301</width>
          <height>211</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="beaker4_lbl">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Beaker 4</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="control_lbl">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Control</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionPreset_2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_Data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPreset">
   <property name="text">
    <string>Preset</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
  <action name="actionPreset_2">
   <property name="text">
    <string>Preset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
